contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer_binary.py

```diff
@@ -19,13 +19,16 @@
 class BuildozerBinary(NativeTool):
   options_scope = 'buildozer-binary'
   name = 'buildozer'
-  # TODO: Move this to bin/buildozer - buildozer is a native binary.
-  support_dir = 'scripts/buildozer'
   default_version = '0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c'
 
   replaces_scope = 'buildozer'
   replaces_name = 'version'
 
+  # TODO: Move this to bin/buildozer - buildozer is a native binary.
```

Comment:
I copied everything from scripts/buildozer to bin/buildozer, so this should no longer be needed. May be worth removing in a separate change though.

Reply:
I'll make a separate change after this gets merged removing the overrides and TODOs!